### ISSUE TITLE
Rename ATD Engineering Review to TPW-TDS Review (LP) (Prod verification Passed)

### ISSUE THREAD

**Issue by atdservicebot** [cityofaustin/atd-data-tech#18335]
### We are updating the process and review comment document for ATD Engineering Review that are performed on DSD folders (SP, C8, C, ZC)

- [x] Rename folder process ATD Engineering Review (51132) to TPW-TDS Review
**Update related folder process comment document:**
- [X] Rename the file name to from ATD Engineering Review Comments to `TPW-TDS Review `
- [X] Update the document header to ~`TPW Transportation Development Services Review`~ `TPW-TDS Review`
![Image](https://github.com/user-attachments/assets/26e88f62-3f6e-4d76-8693-81c841148854)

----
### DTS Request Submittal:
----

> What application are you using?

AMANDA

> Describe the problem.

The Master Comment Report for "ATD Engineering Review" needs to be updated to reflect the new department.  We'd like the title to read "TPW Planning Review" for zoning, site plan and subdivision applications. 

> Describe the outcome you'd like to see when this feature is implemented.

The headers will be consistent across applications.  

> Describe any workarounds you currently have in place or alternative solutions you've considered.

We are updating the header ourselves.  

> Requested By
Amber H.

**Comment by TracyLinder** [comment]
Matiur would like to keep this in the backlog until he's reviewed this with his bigger team.

**Comment by TracyLinder** [comment]
From Matiur: 

> Good morning Tracy, we had a meeting about the discipline name in AMANDA. Based on the discussion, ATD Engineering Review shall be replaced with TPW-TDS Review.

**Comment by Lpriya-LIT** [comment]
<img width="1374" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBOTgxQ0E9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--49fecd5ef2d339ff11dda5219b115e5d8e81771c/image.png" alt="image.png" />

**Comment by Lpriya-LIT** [comment]
Question : The default assigned user for this process is "ATD Engineering Review" Please let me know whether that has to be chnaged to "TPW - TDS Review" as well 
@Nadin-Nader @TracyLinder

**Comment by Lpriya-LIT** [comment]
@Nadin-Nader @TracyLinder 
The default assigned user for this process is "ATD Engineering Review" Please let me know whether that has to be chnaged to "TPW - TDS Review" as well

**Comment by TracyLinder** [comment]
> The default assigned user for this process is "ATD Engineering Review" Please let me know whether that has to be chnaged to "TPW - TDS Review" as well

@Lpriya-LIT leave it as-is for now; I think this change would need to be communicated by their management to employees so they can find their work. We will do it on another card.

**Comment by Lpriya-LIT** [comment]
@Nadin-Nader Update - Committed changes and requested for Dev and Test Deployment, SCTASK0479410

**Comment by Nadin-Nader** [comment]
Waiting on CTM to deploy

**Comment by Lpriya-LIT** [comment]
Update - The Process and the document has been renamed , Please refer attached screenshot. 
@TracyLinder Please let me know whether I can promote this to Prod 
@Nadin-Nader

**Comment by TracyLinder** [comment]
We still need confirmation to go to PROD.

**Comment by TracyLinder** [comment]
This is approved to promote to PROD.

**Comment by Lpriya-LIT** [comment]
Update - Created Prod Deployment requet , SCTASK0487621

**Comment by TracyLinder** [comment]
@Lpriya-LIT found that the document hasn't been updated in PROD and will need to work with CTM to fix this.

**Comment by TracyLinder** [comment]
Update Distribution (51105)
Initial Distribution (51070)

**Comment by TracyLinder** [comment]
### Update
- The documents have been renamed in prod however the change does not apply to the SP folder b/c the SP folder is using a unique doc.
- I instructed Priya to update the SP doc; she will send DSD an email to be sure they're ok with us changing the name.

**Comment by Lpriya-LIT** [comment]
Update - Sent email to DSD team , waiting on response

**Comment by Lpriya-LIT** [comment]
<img width="1206" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBMGtEQ1E9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--0f11732ddf1a47f4accdb02a124f2ef58afffa27/image.png" alt="image.png" />

**Comment by Lpriya-LIT** [comment]
Update - The document name has been updated for SP folder, Please refer screenshot . Ready for UAT on Test

**Comment by TracyLinder** [comment]
@MirUlfath as discussed please analyze the history of this request and identify the UAT blocker (it could be me), and get it moving forward.

**Comment by TracyLinder** [comment]
### UAT Failed
- The document header should read **TPW Transportation Development Services Review**
FYI @Nadin-Nader @Lpriya-LIT 

![Image](https://github.com/user-attachments/assets/141b98d2-2c5d-4af4-80df-fd92501f51e6)

**Comment by Lpriya-LIT** [comment]
@TracyLinder The document name is "TPW Transportation Development Services Review Comments". Please refer the attached screenshot.
The text that appears in the generated document is the process name. In AC , it is mentioned that the process name has to be "TPW-TDS Review". 
Please let me know whether the process name has to be changed to "TPW Transportation Development Services Review". If we change the process name the document will display that.

**Comment by Nadin-Nader** [comment]
@TracyLinder Do we just go with the acronyms or spell it out?

**Comment by TracyLinder** [comment]
Hi @Lpriya-LIT the only thing that needs to be fixed is the title that appears at the top of the generated document. The AC is correct.

**Comment by Lpriya-LIT** [comment]
<img width="1237" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBN2ZhQ1E9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--ffaf2c445b5f24986ca812adec95ca52fd80ddff/image.png" alt="image.png" />

**Comment by Lpriya-LIT** [comment]
Update - Added document title , Ready for UAT on Test

**Comment by MirUlfath** [comment]
Scheduled UAT with Matiur today, 05/01/2025

**Comment by MirUlfath** [comment]
@Lpriya-LIT 

UAT performed by Matiur. UAT Failed.

1. When tested, the Document Type displays "TPW Transportation Development Services Review". Instead, the document Type must display "TPW-TDS Review Comments"
![Image](https://github.com/user-attachments/assets/722423b7-2b1c-4415-ad66-f7edfdc7ec78)



2. When tested, the TDS Comments document header displays the following "TPW-TDS Review".
![Image](https://github.com/user-attachments/assets/e7c08343-0eb4-4117-8925-13beb780514d)

**Instead, the document header must display "TPW Transportation Development Services Review". (see below)**
![Image](https://github.com/user-attachments/assets/b071b799-9e7d-4032-aef9-332a9bcb3f31)

**Comment by MirUlfath** [comment]
Had a conversation with Priya about the issues above. Priya mentioned that #1 can be updated. Whereas the header on #2 is pulling in the Process Name. Hence TPW-TDS Review shows as the header. I will inform the user and confirm what they would like it to be.

**Comment by TracyLinder** [comment]
Matiur said "TPW-TDS Review" will be ok. FYI @MirUlfath

**Comment by Lpriya-LIT** [comment]
<img width="1322" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBODc3Q1E9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--d4e088b469388d7be074e072abdd5868f52acd62/image.png" alt="image.png" />

**Comment by Lpriya-LIT** [comment]
@MirUlfath I have updated the document name , Please refer the attached screenshots.
As I mentioned earlier , all review documents in SP has the folder type as prefix. So I have updated teh name as "SP - TPW-TDS Review Comments" for SP folder and for the other folder types as "TPW-TDS Review Comments".

**Comment by MirUlfath** [comment]
Scheduled UAT Session with Amber and Matiur for Thursday

**Comment by MirUlfath** [comment]
@Lpriya-LIT 

The document name and header works as requested. However, the highlighted text shows above the header on C8, C, and ZC that should not display. Did we add that recently or did that already exist? If we added it, it needs to be removed.
![Image](https://github.com/user-attachments/assets/70e03e02-88ae-4f3c-ae97-5c841773f400)
![Image](https://github.com/user-attachments/assets/3c2a3ed9-62df-41c3-b0cc-95963d2a6b15)
![Image](https://github.com/user-attachments/assets/6a10b67e-1ded-4733-8f6d-03ef55c2f336)

**Comment by Lpriya-LIT** [comment]
@MirUlfath The document header has been removed

**Comment by MirUlfath** [comment]
UAT completed with Amber Hutchens and it passed.

@Lpriya-LIT

**Comment by TracyLinder** [comment]
Hi @Lpriya-LIT which environment is this work in today?

**Comment by Lpriya-LIT** [comment]
@TracyLinder The changes are deployed to Prod , Ready for UAT on Prod

**Comment by MirUlfath** [comment]
UAT was rescheduled to 05/30/2025 with Matiur.

The document name and header on the SPs appear to be correct, but unsure whether the document header was manually updated to display "TPW-TDS Review." Since Matiur can't create a test version in production, I asked him to inform his team to ensure that future documents they generate display the correct name.

We're currently awaiting verification from Matiur's team.

**Comment by MirUlfath** [comment]
Production verification with Matiur and Bryan:

@Lpriya-LIT - The Document Name and Header Name seem to be "TPW-TDS Review" on SP, ZC, C, and C8. However, the document name for ZC is missing the prefix "ZC-" and is not consistent with all the other document names. 

@TracyLinder Tracy - Matiur mentioned that there is another ticket for auto creation of Document Name in ZC so they do not have to manually create one. or another option is to simply rename the following Document Type (10536) to "ZC TPW-TDS Review Comments". As per my conversation with Priya, renaming the following Doc requires less effort.

<img width="347" alt="Image" src="https://github.com/user-attachments/assets/34ffd137-960d-4f94-953c-95b22edcba8a" />
